<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>w_scr_dish_restaurant_table_dish_transfer</class>
 <widget class="QDialog" name="w_scr_dish_restaurant_table_dish_transfer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButton_cur">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="pushButton_des">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="lds_tableView" name="tableView_cur"/>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../res.qrc">:/image/widgets_pic/arrowright.png</pixmap>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="lds_tableView" name="tableView_des"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="lds_toolbar_button" name="pushButton_cur_selectall">
       <property name="text">
        <string>全选</string>
       </property>
       <property name="outer_stylesheet" stdset="0">
        <string>pushbutton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="lds_toolbar_button" name="pushButton_cur_unselectall">
       <property name="text">
        <string>全不选</string>
       </property>
       <property name="outer_stylesheet" stdset="0">
        <string>pushbutton</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_tranfer_print">
       <property name="text">
        <string>厨打</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_tranfer_out">
       <property name="text">
        <string>出品</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="lds_toolbar_button" name="pushButton_ok">
       <property name="text">
        <string>确定</string>
       </property>
       <property name="outer_stylesheet" stdset="0">
        <string>pushbutton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="lds_toolbar_button" name="pushButton_cancel">
       <property name="text">
        <string>取消</string>
       </property>
       <property name="outer_stylesheet" stdset="0">
        <string>pushbutton</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>lds_toolbar_button</class>
   <extends>QPushButton</extends>
   <header>lds_toolbar_button.h</header>
  </customwidget>
  <customwidget>
   <class>lds_tableView</class>
   <extends>QTableView</extends>
   <header>lds_tableview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
